src/app/loginComponents/state/requests.service.ts

```diff
@@ -186,4 +189,22 @@ export class RequestsService {
         }
       );
   }
+
+  deleteOrganization(id: number): void {
+    this.openApiOrgService
+      .deleteRejectedOrPendingOrganization(id)
+      .pipe(finalize(() => this.requestsStore.setLoading(false)))
```

Comment:
when was this set to loading true?

Reply:
loading is set to true by default when Akita initializes the store.

Reply:
Hmm... makes sense that Akita has it set to true when Akita initializes, but something is going to set it to false, even this code.... Even this code sets it to false, what happens if you delete a second org?

Reply:
I'm able to delete multiple orgs successfully. Currently investigating where else setLoading() is set to true aside from the store initialization.

Reply:
Had the wrong impression that the setLoading() flag blocks the store from updating with a new state. Though setLoading() is not set to true anywhere, so I've set the flag to true at the start of the method in the case that we may utilize the flag for future features related to deleting an organization.